requests/adapters.py

```diff
@@ -451,7 +451,7 @@ def send(self, request, stream=False, timeout=None, verify=True, cert=None, prox
                         timeout=timeout
                     )
                 except LocationParseError as e:
-                    raise InvalidURL(e, request=request)
+                    raise InvalidURL(*e.args, request=request)
```

Comment:
This hides the underlying urllib3 `LocationParseError` and is consistent with [requests/models.py:384](https://github.com/psf/requests/blob/9cae7be94f0aa35a26b42e1a34e26f874cd754da/requests/models.py#L384)